lib/src/chart/bar_chart/bar_chart.dart

```diff
@@ -86,7 +86,7 @@ class BarChartState extends AnimatedWidgetBaseState<BarChart> {
       },
       child: CustomPaint(
         key: _chartKey,
-        size: getDefaultSize(context),
+        size: chartSize,
```

Comment:
Sorry for the delay.
Please do not change this line, we set the default size it means if it has not any size constrains, it uses this default size (It will be ignored if we provide a size for it) then let it be like before, it is working properly.

Reply:
Okey i will remove that. I cant do it in the following days. After monday